apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/page.tsx

```diff
@@ -1,3 +1,57 @@
-export default function ConnectedEvaluationsPage() {
-  return null // --> layout.tsx
+import { paginateQuery } from '@latitude-data/core/lib/index'
+import { QueryParams } from '@latitude-data/core/lib/pagination/buildPaginatedUrl'
+import { computeEvaluationResultsWithMetadataQuery } from '@latitude-data/core/services/evaluationResults/computeEvaluationResultsWithMetadata'
+import { findCommitCached } from '$/app/(private)/_data-access'
+import { ROUTES } from '$/services/routes'
+
+import { EvaluationResults } from './_components/EvaluationResults'
+import { fetchEvaluationCached } from './_lib/fetchEvaluationCached'
+
+function pageUrl(params: {
+  projectId: string
+  commitUuid: string
+  documentUuid: string
+  evaluationId: string
+}) {
+  return ROUTES.projects
+    .detail({ id: Number(params.projectId) })
+    .commits.detail({ uuid: params.commitUuid })
+    .documents.detail({ uuid: params.documentUuid })
+    .evaluations.detail(Number(params.evaluationId)).root
+}
+
+export default async function ConnectedEvaluationPage({
+  params,
+  searchParams,
+}: {
+  params: {
+    projectId: string
+    commitUuid: string
+    documentUuid: string
+    evaluationId: string
+  }
+  searchParams: QueryParams
```

Comment:
So before this changes everthing was in the layout and create batch modal was route URL. This is not possible because query params (search params) are not revalidated in layouts so I need to convert results table in a page

More info:
https://nextjs.org/docs/app/api-reference/file-conventions/layout#layouts-do-not-receive-searchparams

Reply:
classic question: is this messing with any modals in subroutes?

Reply:
Yes, create batch was a route and now is not because if we keep as route the table is not visible when open